app/src/main/res/layout/navigation_drawer_header.xml

```diff
@@ -1,4 +1,4 @@
-<android.support.percent.PercentFrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<com.kabouzeid.gramophone.views.WidthFitSquareLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this necessary for the statusbar to work?

Reply:
I think not, its just that album arts are squares and the header is a rectangle. With a transparent status bar that was drawing over it, it looked ok, but now with a solid status bar and displaying the entire artwork below the wrong aspect ratio really pops. Artworks with text look really bad as part of the face or text is just cut off. 

Pushing the header below the status bar was necessary, making it square was to fix its apperance (personal preference). Feel free to adjust it however you like.